Resources/Prototypes/SimpleStation14/Catalog/Cargo/cargo_science.yml

```diff
@@ -0,0 +1,23 @@
+- type: cargoProduct
+  id: ScienceMaterials
+  name: Science materials crate
+  description: "A crate containing a variety of raw materials relevant to research and development"
+  icon:
+    sprite: Nyanotrasen/Structures/Machines/glimmer_machines.rsi
+    state: base
+  product: CrateScienceMaterials
+  cost: 6500
+  category: Science
+  group: market
+
+- type: cargoProduct
+  id: ScienceAPE
+  name: APE crate
+  description: "A crate containing a single APE unit for containing and poking anomalies"
```

Comment:
Poking?

Reply:
poking/prodding with particles for research